src/argbash.m4

```diff
@@ -137,26 +133,36 @@ settle_wrapped_fname()
 	# Based on http://stackoverflow.com/a/19772067/592892
 	local _srcfiles=() _file_found _found ext srcstem searchdir line stem="$2"
 	while read -r line; do
-		_srcfiles+=("$line")
-	done < <(echo 'm4_changecom()m4_define([ARGBASH_WRAP])' "$(cat "$1")" \
-			| autom4te -l m4sugar -t 'ARGBASH_WRAP:$1' 2> "$discard")
+		_srcfiles+=("${line}")
+	done < <(echo 'm4_changecom()m4_define([ARGBASH_WRAP])' "$(cat "$1")" |
+		autom4te -l m4sugar -t 'ARGBASH_WRAP:$1' 2> "${discard}")
 
 	test "${#_srcfiles[@]}" -gt 0 || return
-	for srcstem in "${_srcfiles[@]}"
-	do
+	for srcstem in "${_srcfiles[@]}"; do
 		_found=no
-		for searchdir in "${_arg_search[@]}"
-		do
-			test -f "$searchdir/$srcstem.m4" && { _found=yes; ext='.m4'; break; }
-			test -f "$searchdir/$srcstem.sh" && { _found=yes; ext='.sh'; break; }
-			test -f "$searchdir/$srcstem" && { _found=yes; ext=''; break; }
+		for searchdir in "${_arg_search[@]}"; do
+			test -f "${searchdir}/${srcstem}.m4" && {
```

Comment:
ditto. Sometimes, the block is preceded by `&&`, sometimes (later) by `||`, and one has to look at the end of the first line to get the context. Please move the operator at the beginning of the line, even if it wastes one line.